monkey/infection_monkey/utils/timer.py

```diff
@@ -27,6 +28,13 @@ def is_expired(self):
         """
         return self.time_remaining == 0
 
+    def update_start_time(self, new_start_time: float):
+        """
+        Update self._start_time to new_start_time
+        :param float new_start_time: New time to which self._start_time should be updated
```

Comment:
If we wanted to know what it does to the code we would look to the code. This should specify that `new_start_time` is in what? Floating point seconds from the epoch? Also, wouldn't it make more sense to just create a new timer?